spec/factory_girl/evaluator_class_definer_spec.rb

```diff
@@ -15,7 +15,7 @@
   end
 
   it "evaluates the block in the context of the evaluator" do
-    dependency_attribute = stub("dependency", name: :dependency, to_proc: -> { 1 })
+    dependency_attribute = double("dependency", name: :dependency, to_proc: -> { 1 })
```

Comment:
Hmmm, this dupe assignment was already there. I'll fix the line-length, but I'm going to ignore the dupe assignment for now.